### ISSUE TITLE
Check for two more logging antipatterns

### ISSUE THREAD

**Issue by jenstroeger** [adamchainz/flake8-logging#185]
### Description

As I’m cleaning up a bunch of code using this delightful plugin (thank you!) I noticed that the plugin doesn’t pick up two particular issues:
```python
logger.info("Something about foo")
logger.info(foo)
```
I see two problems with this code snippet:

1. Consecutive logger calls should be consolidated (makes reading logs a pain when these lines are separated); and
2. The format string `msg` ([docs](https://docs.python.org/3/library/logging.html#logging.Logger.debug)) is missing.

Would be great to have the plugin check for these two code smells as well!

<hr>

**Addendum:** looks like empty argument list isn’t flagged either, e.g.
```python
logger.info()  # This fails at runtime.
```
Addendum: I just came across this logging statement which wasn’t flagged by ruff either:
```python
logger.debug("")  # Runs but warrants a warning.
```
